network/transport/v2/handlers.go

```diff
@@ -238,11 +245,8 @@ func (p *protocol) handleGossip(peer transport.Peer, envelope *Envelope) error {
 		}
 	}
 	refs = refs[:i]
-	if len(refs) > 0 {
-		// TODO swap for trace logging
-		log.Logger().Infof("received %d new transaction references via Gossip", len(refs))
-		p.gManager.GossipReceived(peer.ID, refs...)
-	}
+	// TODO swap for trace logging
+	log.Logger().Infof("received %d new transaction references via Gossip", len(refs))
```

Comment:
fix TODO?

Reply:
There are many handlers with the same TODO. We should all update them at the same time, but maybe wait until we are done tweaking the network protocol.